### PR TITLE
Jere style changes

### PR DESCRIPTION
**Pull request by JerebChase**
Configuration page style changes.

This also addresses #48 and #37

### CODE REVIEW

File: app/templates/users.hbs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <div class='top-bar'>
     Users
 
-    {{search-bar searchParameter=fuzzySearchParams.parameter placeholder="User name" onInput=(action "onFuzzyUserSearch")}}
+    {{search-bar searchParameter=fuzzySearchParams.parameter placeholder="Search Users" onInput=(action "onFuzzyUserSearch")}}
```
Comment (ekraihan) [inline app/templates/users.hbs]: We don't currently use fuse to search our users.  We use a stored proc that searches only on user name.. Thats why I had kept "User name." :) But we can of course switch to fuse

**Comment by ekraihan** [comment]
@JerebChase one other thing... I still personally feel like the space between the main header and the column headers on the config page are too far apart... @amenders any tots?

**Comment by amenders** [comment]
I'll check this afternoon. I can't see now, but is this thing pointing at master? That's a pretty long commit summary...

Amos Endersen
(317) 600-9236

File: app/templates/users.hbs (hunk 1 above)
Comment (JerebChase) [reply app/templates/users.hbs]: I don't mind changing the search bar to say Search User Name. And I'll mess with spacing...

**Comment by JerebChase** [comment]
I'll mess with the spacing and let you know when that's done.

**Pull request merged**